test/uk/gov/hmrc/bindingtariffclassification/migrations/MigrationRunnerTest.scala

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.migrations
+
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{atLeastOnce, never, verify, when}
+import uk.gov.hmrc.bindingtariffclassification.base.BaseSpec
+import uk.gov.hmrc.bindingtariffclassification.model.JobRunEvent
+import uk.gov.hmrc.bindingtariffclassification.repository.MigrationLockRepository
+import util.TestMetrics
+
+import java.time.ZonedDateTime.now
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future.{failed, successful}
+
+class MigrationRunnerTest extends BaseSpec {
+
+  class Test {
+    private def jobName                              = "MigrationJobTest"
+    val migrationJob: MigrationJob                   = mock[MigrationJob]
```

Comment:
the base spec extends MockitoSugar, do you think will be too much work to remove it in line with the other tickets we are playing?